src/states/infiniteweightpeps.jl

```diff
@@ -1,11 +1,11 @@
 
 """
-    const PEPSWeight{S}
+    const PEPSWeight
 
 Default type for PEPS bond weights with 2 virtual indices, conventionally ordered as: ``wt : WS ← EN``. 
 `WS`, `EN` denote the west/south, east/north spaces for x/y-weights on the square lattice, respectively.
 """
-const PEPSWeight{T,S} = AbstractTensorMap{T,S,1,1} where {T<:Number,S<:ElementarySpace}
+const PEPSWeight = DiagonalTensorMap
```

Comment:
```suggestion
const PEPSWeight{T,S} = AbstractTensorMap{T,S,1,1}
```
I think we can just keep this as is, while still using DiagonalTensorMap under the hood?

Reply:
I didn't read very carefully... personally I do want to further restrict the weights to be `DiagonalTensorMap`s, which can also help catch implementation errors. In my use cases they are always singular value spectrums. Do you have something else in mind that may one day require the bond weights to go beyond the singular values?

Reply:
I think there is no real downside to keeping it as an `AbstractTensorMap{T,S,1,1}`, right? The type is slightly less restrictive and would work just the same. Perhaps there can also be use cases where the PEPS weights don't come out of an SVD necessarily?

Reply:
Well then I'll just privately make sure that the weights produced by time evolution are always DiagonalTensorMaps 😄